delegate_main.h

```diff
@@ -114,20 +122,14 @@ class Delegate {
   void CreateCacheOp(const OpData& op_data);
 
   std::vector<std::shared_ptr<tim::vx::Operation>>& GetOps() { return ops_; }
+  struct OperationDataType GetOp_info() { return op_info_; }
```

Comment:
GetOperationOutput( index : uint32_t) : uint32_t

return index in tflite.

Reply:
done